### DISCUSSION TITLE
MACS3 vs MACS2

### DISCUSSION THREAD

**Discussion by MarkoBarovic** [GreenleafLab/ArchR#1908] [Questions / Documentation]
Hello! First of all thank you for this great package and great vignette and explanations!

I was wondering if there is an option to run MACS3 instead of MACS2 for peak calling? Would there be any caveats to that? If yes, would there be something different in the handling or do I just point ArchR to the MACS3 directory?

Thanks in advance!

**Comment by rcorces** [comment]
This is not currently an option.
